pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/BitmapInvertedIndexReader.java

```diff
@@ -95,32 +90,32 @@ private synchronized ImmutableRoaringBitmap buildRoaringBitmapForIndex(final int
     final int bufferLength = nextOffset - currentOffset;
 
     // Slice the buffer appropriately for Roaring Bitmap
-    ByteBuffer bb = buffer.toDirectByteBuffer(currentOffset, bufferLength);
+    ByteBuffer bb = _buffer.toDirectByteBuffer(currentOffset, bufferLength);
     ImmutableRoaringBitmap immutableRoaringBitmap = null;
     try {
```

Comment:
Recommend removing this try-catch. Returning null will make it even harder to debug

Reply:
Done.